platform/windows/godot_res_wrap.rc

```diff
@@ -21,7 +17,7 @@ BEGIN
             VALUE "FileVersion",            VERSION_NUMBER
             VALUE "ProductName",            VERSION_NAME " (Console)"
             VALUE "Licence",                "MIT"
-            VALUE "LegalCopyright",         "Copyright (c) 2007-" _MKSTR(VERSION_YEAR) " Juan Linietsky, Ariel Manzur and contributors"
+            VALUE "LegalCopyright",         "(c) 2007-present Juan Liniestky, Ariel Manzur and Godot Engine contributors"
```

Comment:
```suggestion
            VALUE "LegalCopyright",         "(c) 2007-present Juan Linietsky, Ariel Manzur and Godot Engine contributors"
```
Unless it's spelled wrong everywhere else?

Reply:
Oops.

Reply:
Fixed, thanks for catching that.

Reply:
I toyed with the idea of just having `(c) 2007-present Godot Engine contributors` to keep it short, but then changed my mind and typed the names back :)